src/bsp/bm_mote_bristleback_v1_0/Core/Src/stm32u5xx_it.c

```diff
@@ -112,6 +112,33 @@ void EXTI8_IRQHandler(void)
   /* USER CODE END EXTI8_IRQn 1 */
 }
 
+/**
+  * @brief This function handles EXTI Line9 interrupt.
+  */
+void EXTI9_IRQHandler(void)
+{
+  /* USER CODE BEGIN EXTI9_IRQn 0 */
+  BaseType_t rval = pdFALSE;
+  /* USER CODE END EXTI9_IRQn 0 */
+  if (LL_EXTI_IsActiveFallingFlag_0_31(LL_EXTI_LINE_9) != RESET)
+  {
+    LL_EXTI_ClearFallingFlag_0_31(LL_EXTI_LINE_9);
+    /* USER CODE BEGIN LL_EXTI_LINE_9_FALLING */
+    rval |= STM32IOHandleInterrupt((const STM32Pin_t *)VUSB_DETECT.pin);
+    /* USER CODE END LL_EXTI_LINE_9_FALLING */
+  }
+  if (LL_EXTI_IsActiveRisingFlag_0_31(LL_EXTI_LINE_9) != RESET)
+  {
+    LL_EXTI_ClearRisingFlag_0_31(LL_EXTI_LINE_9);
+    /* USER CODE BEGIN LL_EXTI_LINE_9_RISING */
+    rval |= STM32IOHandleInterrupt((const STM32Pin_t *)VUSB_DETECT.pin);
+    /* USER CODE END LL_EXTI_LINE_9_RISING */
+  }
+  /* USER CODE BEGIN EXTI9_IRQn 1 */
+  portYIELD_FROM_ISR(rval);
+  /* USER CODE END EXTI9_IRQn 1 */
+}

```

Comment:
this interrupt will call `vusbDetectIRQHandler` from `usb.c:30` for every rising/falling event detected on VUSB_DETECT.